game/i18n.cc

```diff
@@ -89,25 +92,27 @@ std::map<std::string, std::string> TranslationEngine::GetAllLanguages(bool refre
 	}
 	if (m_languages.size() != 0) return m_languages;
 
+	// Internally, all strings are kept in English, but they are eventually
+	// displayed as menu entries thus they need translation
 	m_languages = {
-		{ "None", _("Auto") },
-		{ "ast_ES.UTF-8", _("Asturian") },
-		{ "da_DK.UTF-8", _("Danish") },
-		{ "de_DE.UTF-8", _("German") },
-		{ "en_US.UTF-8", _("English") },
-		{ "es_ES.UTF-8", _("Spanish") },
-		{ "fa_IR.UTF-8", _("Persian") },
-		{ "fi_FI.UTF-8", _("Finnish") },
-		{ "fr_FR.UTF-8", _("French") },
-		{ "hu_HU.UTF-8", _("Hungarian") },
-		{ "it_IT.UTF-8", _("Italian") },
-		{ "ja_JP.UTF-8", _("Japanese") },
-		{ "nl_NL.UTF-8", _("Dutch") },
-		{ "pl_PL.UTF-8", _("Polish") },
-		{ "pt_PT.UTF-8", _("Portuguese") },
-		{ "sk_SK.UTF-8", _("Slovak") },
-		{ "sv_SE.UTF-8", _("Swedish") },
-		{ "zh_CN.UTF-8", _("Chinese") }
+		{ "None", translate_noop("Auto") },
+		{ "ast_ES.UTF-8", translate_noop("Asturian") },
+		{ "da_DK.UTF-8", translate_noop("Danish") },
+		{ "de_DE.UTF-8", translate_noop("German") },
+		{ "en_US.UTF-8", translate_noop("English") },
+		{ "es_ES.UTF-8", translate_noop("Spanish") },
+		{ "fa_IR.UTF-8", translate_noop("Persian") },
+		{ "fi_FI.UTF-8", translate_noop("Finnish") },
+		{ "fr_FR.UTF-8", translate_noop("French") },
+		{ "hu_HU.UTF-8", translate_noop("Hungarian") },
+		{ "it_IT.UTF-8", translate_noop("Italian") },
+		{ "ja_JP.UTF-8", translate_noop("Japanese") },
+		{ "nl_NL.UTF-8", translate_noop("Dutch") },
+		{ "pl_PL.UTF-8", translate_noop("Polish") },
+		{ "pt_PT.UTF-8", translate_noop("Portuguese") },
+		{ "sk_SK.UTF-8", translate_noop("Slovak") },
+		{ "sv_SE.UTF-8", translate_noop("Swedish") },
+		{ "zh_CN.UTF-8", translate_noop("Chinese") }
```

Comment:
Please refrain from using `translate_noop`. Use `_("you english translation key")` instead

`translate_noop` is solely used within the request_provider to provide the webfront-end its translations. Rethinking that, that should also use `_()` since they're just static strings

Reply:
The problem is that I don't want the string to be translated here, I want it to have a translation that will be used later (when creating the menu). translate_noop seems to be the way yo do this as _() always produces a translated string. That said, I did not understood the problem with usage of this macro...